javascript/src/URDFLoader.js

```diff
@@ -271,6 +276,45 @@ class URDFLoader {
             obj.colliders = colliderMap;
             obj.visual = visualMap;
 
+            // Link up mimic joints
+            jointList.forEach(j => {
+
+                if (j.mimicJoint) {
```

Comment:
Lets use `j instanceof URDFMimicJoint` to be robust to the API changing a bit.

Reply:
Good idea